packages/dapp-connector/test/WalletApi/Cip30Wallet.test.ts

```diff
@@ -43,11 +43,11 @@ describe('Wallet', () => {
 
   it('should return initial api as plain javascript object', () => {
     // Verbose to enable easy detection of which are missing
-    expect(Object.hasOwn(wallet, 'apiVersion')).toBe(true);
-    expect(Object.hasOwn(wallet, 'enable')).toBe(true);
-    expect(Object.hasOwn(wallet, 'icon')).toBe(true);
-    expect(Object.hasOwn(wallet, 'isEnabled')).toBe(true);
-    expect(Object.hasOwn(wallet, 'name')).toBe(true);
+    expect(wallet.hasOwnProperty('apiVersion')).toBe(true);
+    expect(wallet.hasOwnProperty('enable')).toBe(true);
+    expect(wallet.hasOwnProperty('icon')).toBe(true);
+    expect(wallet.hasOwnProperty('isEnabled')).toBe(true);
+    expect(wallet.hasOwnProperty('name')).toBe(true);
```

Comment:
Let's also add `supportedExtensions`. It's also part of the initial API

Reply:
Good catch, added in c43980223a7cccdaaaa7252159e74e4b31e6705d